Java/src/main/java/models/ClassificationLevel.java

```diff
@@ -1,5 +1,5 @@
 package models;
 
 public enum ClassificationLevel {
-    UNCLASSIFIED, CONFIDENTIAL, SECRET, TOP_SECRET
+    UNCLASSIFIED, SECRET, TOP_SECRET
```

Comment:
It can also implement some general interface shared with types and then reuse the same methods